### PR TITLE
Article about subscription renewals

### PR DESCRIPTION
**Pull request by jacegu**
We were missing a detailed explanation on how we handle renewals and specially renewal failures. This is my attempt to fix that.

### CODE REVIEW

**Comment by jacegu** [comment]
I have the feeling that we can skip the email images. They don't add too much info, as the email will be rendered differently in different clients and the fact that it's not clickable text with similar colors to our own palette is confusing.

I will leave them there for now until you share your thoughts :)

**Comment by jacegu** [comment]
Decided to remove the images and tweaked the article further.

File: content/articles/subscription-renewals.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Subscription renewals
```
Comment (weppos) [inline content/articles/subscription-renewals.markdown]: Should it be singular?

Comment (jacegu) [reply content/articles/subscription-renewals.markdown]: No strong opinion on this one. In my head this is plural like _Subscriptions_...
I say we delegate the decision to a native 🇺🇸 speaker.

Comment (coreyhaines) [reply content/articles/subscription-renewals.markdown]: I'd probably leave it plural. The article is about the topic, and plural generally represents the concept

File: content/articles/subscription-renewals.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+</note>
+
+## What happens when my subscription is renewed
+
+When the renewal of your subscription is successfully processed, we will send you an invoice and the service will be extended for the duration of your plan's billing cycle: either 1 month or 1 year.
+
+
+## What happens when the payment for the renewal fails
+
+If your subscription renewal payment fails, you will receive a dunned invoice and an email that will include details about the failure.
```
Comment (coreyhaines) [inline content/articles/subscription-renewals.markdown]: Would it be worth linking out to an explanation of "dunned" ?

Comment (itsalyse) [reply content/articles/subscription-renewals.markdown]: This is a good idea

File: content/articles/subscription-renewals.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+If this happens please don't panic: your account will not be canceled; and, your domains will continue to resolve. We are well aware that credit cards expire and wallets are lost; that's why we give our customers plenty of time to resolve their payment issues. Since the first failed payment attempt **you have 20 days to finalize the payment**.
```
Comment (itsalyse) [inline content/articles/subscription-renewals.markdown]: Should be: "your account will not be canceled, and your"

File: content/articles/subscription-renewals.markdown (hunk 1 above)
Comment (itsalyse) [reply content/articles/subscription-renewals.markdown]: Should be plural

File: content/articles/subscription-renewals.markdown [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+# Subscription Renewals
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Depending on the billing cycle of the plan for which you signed up, your subscription will be renewed either monthly or yearly. The payment for the renewal will be charged to the card on file or, in case you have an a subscription credit, deducted from from the credit balance.
```
Comment (itsalyse) [inline content/articles/subscription-renewals.markdown]: change 'in case' to 'if'

File: content/articles/subscription-renewals.markdown [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+## What happens when my subscription is renewed
+
+When the renewal of your subscription is successfully processed, we will send you an invoice and the service will be extended for the duration of your plan's billing cycle: either 1 month or 1 year.
+
+
+## What happens when the payment for the renewal fails
+
+If your subscription renewal payment fails, you will receive a dunned invoice and an email that will include details about the failure.
+
+If this happens please don't panic: your account will not be canceled, and your domains will continue to resolve. We are well aware that credit cards expire and wallets are lost; that's why we give our customers plenty of time to resolve their payment issues. Since the first failed payment attempt **you have 20 days to finalize the payment**.
```
Comment (itsalyse) [inline content/articles/subscription-renewals.markdown]: wallets 'get' lost

File: content/articles/subscription-renewals.markdown [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+## What happens when my subscription is renewed
+
+When the renewal of your subscription is successfully processed, we will send you an invoice, and the service will be extended for the duration of your plan's billing cycle: either 1 month or 1 year.
+
+
+## What happens when the payment for the renewal fails
+
+If your subscription renewal payment fails, you will receive a dunned invoice and an email that will include details about the failure.
+
+If this happens, please don't panic. Your account will not be canceled, and your domains will continue to resolve. We are well aware that credit cards expire and wallets are lost; that's why we give our customers plenty of time to resolve their payment issues. Since the first failed payment attempt **you have 20 days to finalize the payment**.
```
Comment (itsalyse) [inline content/articles/subscription-renewals.markdown]: should be: wallets get lost 
and: From the first failed payment attempt

File: content/articles/subscription-renewals.markdown [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+- The number of extra domains you have in your account; and
+- The number of domains that have email forwards configured.
+
+<note>
+If you want to know the exact amount you will be charged in your next renewal, and when your subscription is due to renew, you can find this information on [the account page](/articles/new-account-page/#plan-breakdown).
+</note>
+
+## What happens when my subscription is renewed
+
+When the renewal of your subscription is successfully processed, we will send you an invoice, and the service will be extended for the duration of your plan's billing cycle: either 1 month or 1 year.
```
Comment (itsalyse) [inline content/articles/subscription-renewals.markdown]: Should be: When your subscription renewal is successfully processed

File: content/articles/subscription-renewals.markdown (hunk 5 above)
Comment (jacegu) [reply content/articles/subscription-renewals.markdown]: 👍

File: content/articles/subscription-renewals.markdown [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- The number of extra domains you have in your account; and
+- The number of domains that have email forwards configured.
+
+<note>
+If you want to know the exact amount you will be charged in your next renewal, and when your subscription is due to renew, you can find this information on [the account page](/articles/new-account-page/#plan-breakdown).
+</note>
+
+## What happens when my subscription is renewed
+
+When the renewal of your subscription is successfully processed, we will send you an invoice and the service will be extended for the duration of your plan's billing cycle: either 1 month or 1 year.
```
[the remark this answers is not in the document]
Comment (jacegu) [reply content/articles/subscription-renewals.markdown]: 👍

File: content/articles/subscription-renewals.markdown [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+</note>
+
+## What happens when my subscription is renewed
+
+When the renewal of your subscription is successfully processed, we will send you an invoice, and the service will be extended for the duration of your plan's billing cycle: either 1 month or 1 year.
+
+
+## What happens when the payment for the renewal fails
+
+If your subscription renewal payment fails, you will receive a dunned invoice and an email that will include details about the failure.
```
[the remark this answers is not in the document]
Comment (jacegu) [reply content/articles/subscription-renewals.markdown]: 👍

File: content/articles/subscription-renewals.markdown (hunk 6 above)
Comment (jacegu) [reply content/articles/subscription-renewals.markdown]: 👍 You fixed this one yourself @itsalyse. Thanks! 😄

File: content/articles/subscription-renewals.markdown (hunk 7 above)
Comment (jacegu) [reply content/articles/subscription-renewals.markdown]: 👍

File: content/articles/subscription-renewals.markdown (hunk 2 above)
Comment (jacegu) [reply content/articles/subscription-renewals.markdown]: Added! We already have an article that explains what different invoice states mean. Thanks for the suggestion.

**Comment by jacegu** [comment]
@itsalyse thanks a lot for your review and changes. 

I've addressed your feedback. Do you mind taking a second look? Thanks!

**Comment by itsalyse** [comment]
Looks good!

**Comment by jacegu** [comment]
Thanks @itsalyse. Publishing it then!

**Pull request merged**